dm/simulator/internal/sqlgen/sqlgen.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sqlgen is the logic for generating different kinds of SQL statements.
+package sqlgen
+
+import (
+	"github.com/pingcap/tiflow/dm/simulator/internal/config"
+	"github.com/pingcap/tiflow/dm/simulator/internal/mcp"
+)
+
+// SQLGenerator contains all the operations for generating SQLs.
+type SQLGenerator interface {
+	// GenTruncateTable generates a TRUNCATE TABLE SQL.
+	GenTruncateTable() (string, error)
```

Comment:
I'll take a look at that later.  Currently I still plan to use the string as the output, because it can fulfill my needs for the project, and it will not introduce extra dependencies.